tests/app-test.js

```diff
@@ -1,10 +1,77 @@
-/* global describe, beforeEach, afterEach, it */
+/* global describe, beforeEach, afterEach */
 import { expect } from 'chai';
+import $ from 'jquery';
+import sinon from 'sinon';
+import it from './it-will';
```

Comment:
maybe we should be explicit here about this being a convergent `it` and name it something else.  it might not be immediately clear to someone from the outside, since calling this `it` makes this look like a run-of-the-mill mocha test suite